home-manager/home-manager

```diff
@@ -221,7 +221,7 @@ function doSwitch() {
 function doListGens() {
     # Whether to colorize the generations output.
     local color="never"
-    if [[ -t 1 ]]; then
+    if [ -z "${NO_COLOR+nocolor}" ] && [[ -t 1 ]]; then
```

Comment:
I would suggest something like:
```suggestion
    if [[ ! -v NO_COLOR && -t 1 ]]; then
```

Reply:
I like it!

Reply:
Done in eb50e97 and 3a3ee59

Reply:
TIL about `-v`. If this is for bash, this should be compatible.

```
home-manager no-color 1 % cat test.sh
#!/bin/sh

[ ! -v NO_COLOR ] && echo y
home-manager no-color 1 % shellcheck -f gcc test.sh
test.sh:3:5: warning: In POSIX sh, unary -v (in place of [ -n "${var+x}" ]) is undefined. [SC2039]
```